spec/features/welcome_spec.rb

```diff
@@ -71,4 +71,71 @@
     end
   end
 
+  describe "visit CONTACT page" do
+    context "goes to Contact page and shows info" do
+      it "shows address" do
+        visit '/contact'
+        page.has_css?('row .col-sm-6 .contact', text: "VISIT")
+      end
+      it "shows map in iframe" do
+        visit '/contact'
+        page.has_css?('row .col-sm-6 iframe', visible: :show)
```

Comment:
this is not really checking that the iframe is working, just that you have one on the page